hax/treemakers/se_treemaker.py

```diff
@@ -0,0 +1,64 @@
+import hax
+import numpy as np
+from pax import units
+
+
+def yield_single_electrons(event, nhits_bounds, width_bounds):
```

Comment:
Can you add a _ before the function since this is "private" to the treemaker?  Or you can move it within the class.

Reply:
Also, add a docstring that explains what the function does